.travis.yml

```diff
@@ -24,5 +24,18 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 6" CHECK=spec
   - rvm: 2.1.9
     env: PUPPET_GEM_VERSION="~> 4" CHECK=rubocop
+  -
+    bundler_args: ""
+    dist: trusty
+    rvm: 2.5
+    before_script:
+    - bundle exec rake 'litmus:provision[docker, centos:7]'
+    - bundle exec rake 'litmus:install_agent[puppet6]'
+    - bundle exec rake litmus:install_module_fixtures
+    - bundle exec rake litmus:install_gems
+    script:
+    - bundle exec rake litmus:acceptance:parallel
```

Comment:
Is there an easy way to run the acceptance tests locally or do we just have to run all 5 rake commands in order? `rake litmus:acceptance:localhost` doesn't seem to find any examples.

Reply:
>Is there an easy way to run the acceptance tests locally

Currently no. 

> do we just have to run all 5 rake commands in order? 

What I'm missing is a README update to explain how to run the Acceptance Tests and why.

Reply:
Added a README change.